docs/src/main/asciidoc/security-basic-authentication-howto.adoc

```diff
@@ -16,7 +16,11 @@ Enable Basic authentication for your Quarkus project and allow users to authenti
 quarkus.http.auth.basic=true
 ----
  
-For a Basic authentication configuration walk-through that uses `JPA`, see the xref:security-basic-authentication-tutorial.adoc[Secure a Quarkus application with Basic authentication] guide.
+The easiest way to configure user credentials required for a Basic authentication to work is to configure the user name, secret and roles directly in `application.properties`, see the xref:security-testing.adoc#configuring-user-information[Configuring User Information] section for more information.
+
+However, configuring the user name, secret and roles in `application.properties` is only suitable for testing and instead, using a database to store this information is recommended.
+
+For a Basic authentication configuration walk-through that uses `JPA` to store user credentials in a database, see the xref:security-basic-authentication-tutorial.adoc[Secure a Quarkus application with Basic authentication] guide.
```

Comment:
```suggestion
To walk through how to configure Basic authentication together with JPA for storing user credentials in a database, see the xref:security-basic-authentication-tutorial.adoc[Secure a Quarkus application with Basic authentication] tutorial.
```

Reply:
@sberyozkin - I know you didn't touch this sentence in this PR but while we are here, should we simplified the phrasing as per above :arrow_up: ?  Thanks .